### PR TITLE
Gives Pocket protectors to various nerdy jobs on the station.

### PR DESCRIPTION
**Pull request by Expletive**
:cl:
add: Nerds on the station will be pleased to know they now have pocket protectors.
/:cl:

sprite: pocketprotector

Worn:
nerdswithprotectedpockets

Time to give nerds the stereotypical nerd fashion they crave. Pocket protectors, to protect your pockets.

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
in what way does it break the game

File: code/modules/clothing/under/accessories.dm [hunk 1]
```diff
@@ -255,6 +255,23 @@
...
+////////////////
+/obj/item/clothing/accessory/pocketprotector
+	name = "pocket protector"
+	desc = "Can protect your clothing from ink stains, but you'll look like a nerd if you're using one."
+	icon_state = "pocketprotector"
+	item_color = "pocketprotector"
+	pockets = /obj/item/weapon/storage/internal/pocket/pocketprotector
+
+/obj/item/clothing/accessory/pocketprotector/full
+	pockets = /obj/item/clothing/accessory/pocketprotector/full
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/under/accessories.dm]: /obj/item/weapon/storage/internal/pocket/pocketprotector/full/

File: code/modules/clothing/under/accessories.dm [hunk 2]
```diff
@@ -255,6 +255,23 @@
...
+
+/obj/item/clothing/accessory/pocketprotector/cosmetology
+	pockets = /obj/item/clothing/accessory/pocketprotector/cosmetology
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/under/accessories.dm]: /obj/item/weapon/storage/internal/pocket/pocketprotector/cosmetology/

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
I probably found it.

**Comment by Expletive** [comment]
Oh, damn. I am an idiot. Created an infinite loop.

**Comment by optimumtact** [comment]
good spot cobby

**Comment by ShadowDeath6** [comment]
I think the protector's sprite could be a bit lower, it's kind of entering the neck area on a lot of the jumpsuit sprites

**Comment by tortellinitony** [comment]
Eww, gradient sprite.

**Comment by octareenroon91** [comment]
I'm curious why roboticists are excluded here.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**